blockchain/agendas_test.go

```diff
@@ -554,8 +415,8 @@ func TestFixedSequenceLocks(t *testing.T) {
 	g.AcceptTipBlock()
 
 	outs = g.OldestCoinbaseOuts()
-	g.NextBlock("b11", nil, outs[1:], replaceFixSeqLocksVersions,
-		chaingen.ReplaceVotes(vbDisapprovePrev, fslVersion),
+	g.NextBlock("b11", nil, outs[1:],
+		chaingen.ReplaceVotes(vbDisapprovePrev, voteVersion),
```

Comment:
This works, but I think it should only be replacing the actual vote bits now instead of mucking about with the voting version.  e.g.

```Go
	chaingen.ReplaceVoteBitsN(0, vbDisapprovePrev),
	chaingen.ReplaceVoteBitsN(1, vbDisapprovePrev),
	chaingen.ReplaceVoteBitsN(2, vbDisapprovePrev),
	chaingen.ReplaceVoteBitsN(3, vbDisapprovePrev),
	chaingen.ReplaceVoteBitsN(4, vbDisapprovePrev),
	func(b *wire.MsgBlock) {
		...
```

Even better would be to introduce a `ReplaceVoteBits(voteBits utint16) func(*wire.MsgBlock)` method on the generator for convenience which works similar to `ReplaceVoteVersions` where it finds the votes and calls `ReplaceVoteBitsN` with the found index and the caller-provided `voteBits`.

That is a generally useful func to have available.

With that in place, the above would reduce to `chaingen.ReplaceVoteBits(vbDisapprovePrev), ...`.

Reply:
Thanks for the suggestion.  I agree that this is a cleaner way to handle it and I added the suggested `ReplaceVoteBits` function to the generator.